src/proto/http.proto

```diff
@@ -0,0 +1,10 @@
+package fedb.http;
+
+option cc_generic_services = true;
+
+message HttpRequest {};
+message HttpResponse {};
+
+service APIService {
```

Comment:
rename to  APIServer and update the api_service_impl file name

Reply:
It's for `brpc::Server::AddService`, `Server` is not a good name.

Reply:
Just follow the naming style like `NameServer` and `TabletServer`

Reply:
We will `StartAPIServer` in fedb.cc, not here

Reply:
1 Our module is called APIServer no APIService. 2 There is no format that if use AddXXX method the variables must be named XXX. 3 Just follow the existing naming style

Reply:
fixed